__tests__/app.test.js

```diff
@@ -148,6 +148,20 @@ describe('GET /api/articles', ()=>{
             expect(body).toBeSortedBy( 'created_at', {descending: true})
         })
     })
+    test('should return articles filtered by topic if topic is included in query', ()=>{
+        return request(app)
+        .get('/api/articles?topic=mitch').then(({body})=>{
+            body.forEach((article)=>{
+                expect(article.topic).toBe('mitch')
+            })
+        })
+    })
+    test('should return no articles i filtered by topic if no article with that topic exists', ()=>{
```

Comment:
The thing is that here not just the articles with that topic, but even the topic ('no_topic') itself doesn't exist.

In this case we would expect 404.

Reply:
Can I push back on this Bela. I wouldn't consider this a 'not found' error just that there is no article with this topic 'yet'... I would just expect an empty search result...